backend/schema/heat_pump.ts

```diff
@@ -0,0 +1,28 @@
+import { FromSchema } from "json-schema-to-ts";
+import { metadataProperties, requiredMetadata } from "./metadata";
+import { coreProperties, requiredCore } from "./core";
+
+const tonnage = ["1", "1.5", "2", "2.5", "3", "3.5", "4", "4.5", "5"];
```

Comment:
I am not 100% sure what the tonnage means here. For instance, does 1.5 mean its between 1-1.5 tons?

Reply:
Tons are not a continuous variable. I left a comment in your Excel schema about this. 

We can play around with it but it’s more like an enum.